src/IntegerPicker/IntegerPicker.js

```diff
@@ -247,7 +247,7 @@ module.exports = kind(
 	*/
 	create: function () {
 		Control.prototype.create.apply(this, arguments);
-		this.verifyValue();
```

Comment:
We no longer need to run `verifyValue`? That sets the value and allows the control to already be set to the value without animating.